gunicorn/CHANGELOG.md

```diff
@@ -1,7 +1,12 @@
 # CHANGELOG - gunicorn
 
-1.0.003-22-2017
-==================
+## 1.1.005-23-2017
```

Comment:
I think this "format" was an error in the script we used. It should be `1.1.0 / 2017-05-23`. We will have to draft a PR to make sure we are consistent across all changelogs. That shouldn't stop you from merging.